EESSI-install-software.sh

```diff
@@ -229,7 +229,8 @@ done
 
 echo ">> Creating/updating Lmod cache..."
 export LMOD_RC="${EASYBUILD_INSTALLPATH}/.lmod/lmodrc.lua"
-if [ ! -f $LMOD_RC ]; then
+lmodrc_changed=$(cat ${pr_diff} | grep '^+++' | cut -f2 -d' ' | sed 's@^[a-z]/@@g' | grep '^create_lmodrc.py$' > /dev/null; echo $?)
+if [ ! -f $LMOD_RC ] || [ ${lmodrc_changed} == '0' ]; then
```

Comment:
We need to find a way to allow people to be able to use this script outside of a PR situation

Reply:
I'm not sure I understand why: what is the use case where end-users or sysadmins would have to re-run this?

Reply:
Not just for them, also for us, in testing this PR I have to manually edit these lines

Reply:
let's follow up on that in #435